spec/record_level/replace_fields_spec.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require 'nokogiri'
+require 'marc'
+require 'byebug'
+require 'marc_cleanup'
+
+RSpec.describe 'replace_fields' do
+  let(:record) { MARC::Record.new_from_hash('fields' => fields) }
+  context 'the contents of both source fields match their respective targets (but the indicators match for only the first)' do
```

Comment:
The context implies to me that one change will succeed and one will not. Could you make the test evaluate one replace_field that succeeds due to `ignore_indicators` being true, and one that fails due to `ignore_indicators` being false?